gammapy/image/models/__init__.py

```diff
@@ -1,3 +1,13 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 """Spatial 2D image models."""
 from .core import *
+
+spatial_models = {
```

Comment:
There is a convention, that module level variables should be all upper case. So just rename `spatial_models` to `SPATIAL_MODELS`.